assets/css/style.scss

```diff
@@ -2,7 +2,7 @@
 # Only the main Sass file needs front matter (the dashes are enough)
 ---
 
-$content-width: 960px;
+$content-width: 100%;
```

Comment:
This undoes the fix from #6254 / #6231, right? I believe we wanted a media query for this?